hack/build-go.sh

```diff
@@ -68,14 +68,14 @@ if [ $# -eq 0 ]; then
             # Skip fuzz tests, as they are not part of regular unit testing
             go ${target} -v -tags "${KUBEVIRT_GO_BUILD_TAGS}" -ldflags "$(kubevirt::version::ldflags)" -race -skip FuzzAdmitter -timeout 15m ./pkg/...
         )
+    elif [ "${target}" = "generate" ]; then
+        ln -s ../../staging/src/kubevirt.io/api/ ${KUBEVIRT_DIR}/vendor/kubevirt.io/api
```

Comment:
What is the failure if we don;'t make the symlink?

Reply:
mockgen fails to resolve dependencies if the symlinks are not in place. To me it looks like it is unable to make use of workspaces.